chapters/lang/lexical-structure.tex

```diff
@@ -260,3 +263,7 @@ \subsubsection*{Escape Sequences}
     \end{tabular}
     \end{center}
 \end{itemize}
+
+\subsection{None Literal}
+
+The keyword none evaluates to successful none term, which is a special value none of type none. It is used to make boolean result as succ none or fail none.
```

Comment:
I'd put none in `\lstinline` for readability, also types are title cased, so `none` type is called `None`